setup.py

```diff
@@ -2,7 +2,7 @@
 
 setup(
     name="descwl_shear_sims",
-    version="0.3.0",
+    version="0.3.1",
```

Comment:
We should read this out of the package. I can do this in another PR.

Reply:
I'll do it